### ISSUE TITLE
double quotes in header for filename

### ISSUE THREAD

**Issue by doziteh** [servocoder/RichFilemanager#78]
This is not a bug but it is an issue, I think worth sharing.
First we had a problem with download of a specific file from RFM. 
Our platform is WAMP 2.5
Turns out the problem was comma in filename. 
We added double quotes in header for filename:
Its easier to explain with code
LocalFilemanager.php line 841 for last version:
`header('Content-Disposition: attachment; filename="' . basename($target_fullpath) . '"');`

That way comma and some other charters wont make problems when downloading.
I suggest to include this in main branch.

Second issue is something not easily repeatable but I will explain our problem and solution that works for us if someone else encounters the same problem.
Under slower connections and when our server is under heavier load we have problems with chunk upload. It simply stops after 2 chunks. No error reported in php_error.log or apaches error.log
We tried to eliminate chunks because for our PHP 5.5 that shouldn't be a problem:
`if (ob_get_level()) {
	ob_end_clean();
}
readfile($target_fullpath);`

That worked better but not for the largest files.

Finally we tried 
`$myInputStream = fopen($target_fullpath, 'rb');
$myOutputStream = fopen('php://output', 'wb');

error_log ($myInputStream . "--" . $myOutputStream, 0);
stream_copy_to_stream($myInputStream, $myOutputStream);

fclose($myOutputStream);
fclose($myInputStream);`

and that works for us for all files tried so far.
As mentioned this is a specific issue...

**Comment by servocoder** [comment]
Thanks for the reporting.

I'm going to include a fix for the first issue in the closest release.

According to the second issue, it's interesting observation. I will try to reproduce it, but regardless of the result I want to be sure that I got your solution correct. From what I understood, the solution suppose to replace a block of code ("actionDownload" method) from:

```
            $chunk_size = 16 * 1024 * 1024;
            // read file by chunks to handle large files
            if ($chunk_size && $file_size > $chunk_size) {
                $handle = fopen($target_fullpath, 'rb');
                while (!feof($handle)) {
                    echo fread($handle, $chunk_size);
                    @ob_flush();
                    @flush();
                }
                fclose($handle);
            } else {
                readfile($target_fullpath);
            }
```

to

```
$myInputStream = fopen($target_fullpath, 'rb');
$myOutputStream = fopen('php://output', 'wb');

stream_copy_to_stream($myInputStream, $myOutputStream);

fclose($myOutputStream);
fclose($myInputStream);
```

Is that what you mean?

**Comment by doziteh** [comment]
Yes, this replacement exactly works for us.
As mentioned we cannot reproduce this every time but it happens.
Definitely has to do something with our installation...
I do not recommend including this in your release but simply wanted to share our experience if someone encounters similar problems.
Thank you for your effort.

**Comment by servocoder** [comment]
Fixed in v2.1.0
Alternative solution of reading file by chunks upon "download" process is linked in PHP connector file.
